### ISSUE TITLE
Add Earwig copyvio check link to AFCH header

### ISSUE THREAD

**Issue by NovemLinguae** [WPAFC/afch-rewrite#146]
I wrote a userscript to do this last night. 

https://en.wikipedia.org/wiki/User:Novem_Linguae/Scripts/AFCTools.js

You could integrate it directly into the tool instead, if you're interested. See spot I highlighted with an orange box in the screenshot below. Since copyvio should be one of the first checks done on a draft, makes sense to have this link front and center, instead of off site, or buried in a drop down menu.

![2021-02-26_025645](https://user-images.githubusercontent.com/79697282/109291848-5821fe80-77de-11eb-99de-54143a03fc91.png)

**Comment by primefac** [comment]
Given that there's already a script (User:The Earwig/copyvios.js) to add the link to the copyvio check to the Tools bar, I see little reason to add it to AFCH as well - it would take me two clicks to get there as opposed to the one currently needed.

**Comment by siddharthvp** [comment]
I think it wouldn't hurt to have this, and users wouldn't need to install an additional script. Also, having this button here might even encourage more users to do a copyvio check before accepting.

**Comment by enterprisey** [comment]
Seems like a fine idea to me too. I'd put it on the second line next to the title, to make it obvious what it's going to do.

In a perfect world, that tool would have an API and we could scrape the percent and put it in our interface (so no need to navigate to another page) but I imagine (wouldn't know) that people usually need to check the whole page, so that's probably not a good path to go down.

**Comment by primefac** [comment]
Yeah, we've discussed that with The Earwig in the past, but since Google only allows us a certain number of checks per day (and they're arguably the best) there's no real way for us to automate checking.

**Comment by earwig** [comment]
Some disorganized thoughts (thanks @enterprisey for the ping):

- I'm not opposed to adding this to the script, even though other ways exist to call it (the submission template itself, my toolbar link, and the MoreMenu gadget).
- However, this particular spot feels too prominent.
- Even if they don't _click_ on the link, just having it as a reminder of "hey, don't forget to check for copyvios" isn't a terrible idea, since it could encourage reviewers to be more careful. A more radical suggestion that ties into this is the "review checklist" idea that's been floated a bit. If the script presented a concise list of the "steps" in a review, the copyvio check link could be included there.
- Automated usage of the tool with Google enabled is a no-go from me. Automated usage with the "links in page" and/or "turnitin" options _might_ be worthwhile. Don't show the resulting percentage, but if it's above the suspicion threshold the script could display a warning with a link to the full check.
- This can still take a while to run, and I don't love the idea of the script displaying a loading icon every time it's opened, or worse, no indication of the status of the check at all.
- It would need to be made clear that if the warning doesn't show up, that doesn't mean the article is free of copyvios. The tool isn't magic.
- For what it's worth, EarwigBot used to copyvio check every draft on submission, but that was when we weren't rate-limited by Google and we had maybe 5% of the submissions we have now.
